lte/gateway/c/oai/lib/pcef/pcef_handlers.cpp

```diff
@@ -208,7 +206,7 @@ void pcef_update_teids(
  * else if they are in [48,57] keep them the same
  * else log an error and return '0'=48 value
  */
-static char _convert_digit_to_char(char digit) {
+char _convert_digit_to_char(char digit) {
```

Comment:
Remove the underscore at the start of the function name
```suggestion
char convert_digit_to_char(char digit) {
```